pyro/nn/auto_reg_nn.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, input_dim, hidden_dim, output_dim_multiplier=1,
 
         if mask_encoding is None:
             # the dependency structure is chosen at random
-            self.mask_encoding = 1 + torch.multinomial(torch.ones(input_dim - 1) / (input_dim - 1),
+            self.mask_encoding = 1 + torch.multinomial(torch.tensor(input_dim - 1).fill_(1.).float(),
```

Comment:
The input probs are no longer normalized. Can you confirm that `torch.multinomial` normalizes internally, so that your change is valid?

Reply:
see [note](https://pytorch.org/docs/stable/torch.html?highlight=multinomial#torch.multinomial)